packages/sdk-redux/tsconfig.json

```diff
@@ -52,7 +52,7 @@
         // "experimentalDecorators": true /* Enables experimental support for ES7 decorators. */,
         // "emitDecoratorMetadata": true /* Enables experimental support for emitting type metadata for decorators. */,
         "forceConsistentCasingInFileNames": true,
-        "lib": ["dom", "es2019", "es2020.promise", "es2020.bigint", "es2020.string"],
```

Comment:
I'm pretty sure the DOM lib here is irrelevant. IDE or the build complained a while ago and then I added it... but it doesn't complain anymore and it doesn't make too much sense. The DOM lib is required by Redux Toolkit when generating React Hooks but that shouldn't affect our TSConfig.